README.md

```diff
@@ -34,7 +34,7 @@ To install PyTorch/XLA nightly build in a new TPU VM:
 ```sh
 pip install --pre torch torchvision --index-url https://download.pytorch.org/whl/nightly/cpu
 # Edit `cp310-cp310` to fit your desired Python version as needed
-pip install 'torch_xla[tpu] @ https://storage.googleapis.com/pytorch-xla-releases/wheels/tpuvm/torch_xla-2.8.0.dev-cp310-cp310-linux_x86_64.whl' \
+pip install 'torch_xla[tpu] @ https://storage.googleapis.com/pytorch-xla-releases/wheels/tpuvm/torch_xla.dev-cp310-cp310-linux_x86_64.whl' \
```

Comment:
When trying to do this, I got an error saying that `ERROR: Invalid wheel filename (wrong number of parts): 'torch_xla.dev-cp310-cp310-linux_x86_64'`

Sorry, I didn't notice earlier that python wheel naming conventions always require a version number https://packaging.python.org/en/latest/specifications/binary-distribution-format/#file-name-convention

Should we leave this as it is?

I have a proposal to upload our packages to a [download.pytorch.org](https://download.pytorch.org/whl/nightly/cpu) and plan to create an RFC for it on pytorch repo